cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -34,7 +34,7 @@
 /**
  * Init command for creating a ballerina project.
  */
-@CommandLine.Command(name = INIT_COMMAND, description = "Create a Init Ballerina project")
```

Comment:
I don't think this command is used now. Tagging @hemikak

Reply:
Removed the `init-command.java` file